src/single-spa-react.js

```diff
@@ -11,11 +11,10 @@ const defaultOpts = {
   ReactDOM: null,
   rootComponent: null,
   loadRootComponent: null,
-  domElementGetter: null,
```

Comment:
domElementGetter was listed twice in this object. Removing the "required" one in favor of the "optional" one